services/article/application/src/main/java/nettee/article/usecase/ArticleLikeDeleteUseCase.java

```diff
@@ -0,0 +1,6 @@
+package nettee.article.usecase;
+
+public interface ArticleLikeDeleteUseCase {
+
+    void deleteArticleLike(String userId, String profileId, String articleId);

```

Comment:
`userId` 대신 `profileId`만 있어도 좋을 것 같습니다.

> 아직은 1 계정 1 프로필이므로, 프로필 단위로 삭제하는 기능만 있어도 좋아 보입니다.
> 나중에 use case가 추가되면 `deleteByUserIdAndArticleId(String userId, String articleId)` 메서드를 추가할 수 있어 보여요.

```suggestion
    void deleteByProfileIdAndArticleId(String profileId, String articleId);
```

Reply:
@merge-simpson 님! 정성스러운 리뷰 감사드립니다 : )
방금 말씀해주신 부분 반영하여 커밋했습니다!
감사합니다!